frontend-service/build.gradle

```diff
@@ -37,12 +36,10 @@ dependencies {
     compile "org.mapstruct:mapstruct-jdk8"
 
     compileOnly "org.mapstruct:mapstruct-processor"
-    apt "org.mapstruct:mapstruct-processor"
```

Comment:
y, the `compileOnly` is required to make IntelliJ work with annotation processor.  I don't know what's the value of `apt` plugin, only generation annotation before java compile?  And with `compileOnly`, it will generate annotation during java compile?

In this case, I think just use `compileOnly` would make sense.

Reply:
The links in the PR description have more information on what the `apt` dependency type gets us. (tldr: better transitive dependency isolation, faster incremental compile in some edge cases)